python/lammps.py

```diff
@@ -1439,6 +1444,17 @@ def create_atoms(self,n,id,type,x,v=None,image=None,shrinkexceed=False):
     return self.lib.lammps_create_atoms(self.lmp, n, id_lmp, type_lmp, x_lmp, v_lmp, img_lmp, se_lmp)
 
   # -------------------------------------------------------------------------
+  
+  # return vector of bonds gathered across procs
+  # count should be 3
+
+  def gather_bonds(self):
+    nbonds = int(self.get_thermo('bonds'))
+    data = ((3*nbonds)*c_int)()
```

Comment:
this should be `self.c_tagint` instead of `c_int`

Reply:
Sorry, I am a bit out of practice. Thanks for all the advice, help and patience with me. I've changed it accordingly.